tests/tools/oeapkman/zlib/test.edl

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/logging.edl" import oe_write_ocall;
+    from "openenclave/edl/fcntl.edl" import *;
+#ifdef OE_SGX
+    from "openenclave/edl/sgx/platform.edl" import *;
+#else
+    from "openenclave/edl/optee/platform.edl" import *;
+#endif
+
+    trusted {
+        public int enc_main(int argc, [user_check] char** argv);
```

Comment:
why user_check?  (which we tell people it's better to not use)

Reply:
Fixed.

Reply:
We cannot currently pass array of strings via edger8r without user_check.